src/my_app/analysis_options.yaml

```diff
@@ -0,0 +1,4 @@
+include: package:very_good_analysis/analysis_options.3.1.0.yaml
```

Comment:
Should we use 4.0.0+1 instead?

Reply:
No, since very_good_cli supports min dart sdk 2.18.0, and 4.0.0 is not compatible with dart sdk 2.18.0 .